src/tests/multihost/admultidomain/test_multidomain.py

```diff
@@ -138,12 +138,10 @@ def test_0002_bz2018432(multihost, newhostname, adjoin):
         domain_list_cmd = multihost.client[0].run_command(
             'sssctl domain-list', raiseonerr=False)
         domain_list = domain_list_cmd.stdout_text.split('\n')
-        domain_list.remove("implicit_files")
-        domain_list = domain_list[:-1]
-        multihost_list = multihost.ad
-        multihost_list = multihost_list[:-1]
-
-        for x in multihost_list:
-            assert x.domainname in domain_list
-
-        assert len(domain_list) == len(multihost_list)
+        if "implicit_files" in domain_list:
+            domain_list.remove("implicit_files")
+        multihost_list = []
+        for item in multihost.ad:
+            multihost_list.append(multihost.ad[item].domainname)
+        multihost_list.pop()
+        assert domain_list in multihost_list
```

Comment:
This assert/comparison odes not look correct to me, domain_list will not be element of multihost_list.
I would use
assert len(domain_list) == len(multihost_list) and sorted(domain_list) == sorted(multihost_list)

Reply:
Changed the logic, no longer using this code.